packages/mui-material/src/ButtonBase/ButtonBase.js

```diff
@@ -364,7 +403,6 @@ const ButtonBase = React.forwardRef(function ButtonBase(inProps, ref) {
     >
       {children}
       {enableTouchRipple ? (
-        /* TouchRipple is only needed client-side, x2 boost on the server. */
```

Comment:
Should we move this comment above `enableTouchRipple` to explain why?

Reply:
I added a comment on the `LazyRipple` class.